CONTRIBUTING.md

```diff
@@ -10,15 +10,35 @@
     - [Contributor license agreement](#contributor-license-agreement)
   - [Quickstart definitions](#quickstart-definitions)
     - [Quickstarts](#quickstarts)
+      - [Fields](#fields)
+      - [Style tips](#style-tips)
+      - [Quickstart description prose template](#quickstart-description-prose-template)
+      - [Quickstart usage](#quickstart-usage)
+      - [Authors](#authors)
+      - [Slug & title fields](#slug--title-fields)
+      - [Summary & descriptions](#summary--descriptions)
+      - [Documentation](#documentation)
+      - [Support Levels](#support-levels)
```

Comment:
sorry I meant were these expected changes?

Reply:
oh yeah, i have a markdown autostyler that adds links to headers in md, kept it in beacuse i thought it would be helpful to update the doc links a little but i can remove it at the discretion of the io team

Reply:
I think it's good if it was missing before